src/io.c

```diff
@@ -220,6 +253,14 @@ Char PEEK_CURR_CHAR(void)
     return *STATE(In);
 }
 
+void IGNORE_REST_OF_LINE(void)
```

Comment:
Any suggestioins for a better name? `SKIP_TO_END_OF_LINE`? `SKIP_LINE`? ...?

Reply:
SKIP_TO_END_OF_LINE sounds good to me

Reply:
Renamed to `SKIP_TO_END_OF_LINE`